src/htmlrenderer.cpp

```diff
@@ -1030,22 +1008,29 @@ void HtmlRenderer::add_media_link(std::string& curline,
 	const std::string& media_url, const std::string& media_title,
 	unsigned int media_count, LinkType type)
 {
+	std::string link_url;
+
 	if (media_url.empty()) {
 		return;
 	}
 
+	// Display media_url except if it's an inline image
+	if (type == LinkType::IMG && media_url.substr(0, 5) == "data:") {
+		link_url = "inline image";
+	} else {
+		link_url = utils::censor_url(utils::absolute_url(url, media_url));
+	}
+
 	std::string type_str = type2str(type);
-	unsigned int link_num = add_link(links,
-			utils::censor_url(utils::absolute_url(url, media_url)),
-			type);
+	unsigned int link_num = add_link(links, link_url, type);
 	std::string output;
 
 	if (!media_title.empty()) {
-		output = strprintf::fmt("[%s %u: %s (%s #%u)]", _(type_str.c_str()),
-				media_count, media_title, _("link"), link_num);
+		output = strprintf::fmt("[%s %u: %s (link #%u)]", _(type_str.c_str()),
+				media_count, media_title, link_num);
 	} else {
-		output = strprintf::fmt("[%s %u (%s #%u)]", _(type_str.c_str()),
-				media_count, _("link"), link_num);
+		output = strprintf::fmt("[%s %u (link #%u)]", _(type_str.c_str()),
+				media_count, link_num);
```

Comment:
`_()` is a macro that expands to `gettext()` (see config.h in the root of the repo). We use this to internationalize the application. This change makes "link" untranslatable, so please revert this.

Reply:
ah, thanks for explaining that. I admit I did look at config.h to try to figure out what `_()` does, but with my C++ being rather rusty (and having never properly learned about macros) ... well ... this happened :sweat_smile: I should have run `man gettext`